### PR TITLE
docs(readme): add types

### PR DESCRIPTION
**Pull request by harish-sethuraman**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Added proper types

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,7 +214,12 @@ module.exports = {
 
 ### `minify`
 
-Type: `Function`
+Type:
+
+```ts
+type minify = function;
```
Comment (harish-sethuraman) [inline README.md:220]: @alexander-akait I'd need help here to find the type of this `minify`

Comment (harish-sethuraman) [reply README.md]: Done

**Comment by alexander-akait** [comment]
Please update snapshots

**Comment by alexander-akait** [comment]
Because deps were update (maybe miss cache on CI, so the latest version was installed for all packages)

**Comment by harish-sethuraman** [comment]
@alexander-akait I ran jest --updateSnapshot but I didn't see any changes in the local? What am I doing wrong? Can you help me out please?

**Comment by harish-sethuraman** [comment]
Done

**Comment by harish-sethuraman** [comment]
Seems like now it is not failing for snapshot but failed due to some other error @alexander-akait

**Comment by alexander-akait** [comment]
let's try to regenerate lock file, thanks

**Comment by harish-sethuraman** [comment]
Do you mean we want to delete the existing file and generate a completely new one?

**Comment by alexander-akait** [comment]
Looks random fail on CI, don't worry, let's merge

**Comment by alexander-akait** [comment]
Big thanks

**Comment by harish-sethuraman** [comment]
😁

**Pull request merged**